src/v2/Apps/Conversation/Components/ConversationList.tsx

```diff
@@ -26,6 +26,16 @@ const SpinnerContainer = styled.div`
   position: relative;
 `
 
+const ConstrainedHeightContainer = styled(Flex)`
+  height: calc(100vh - 103px);
```

Comment:
I wanted to avoid repeating this line of code [from a parent component](https://github.com/artsy/force/blob/main/src/v2/Apps/Conversation/Routes/Conversation/index.tsx#L28-L33), especially having a number in `px` in it. I tried extending the component in styled-components (example below), and the result is two different classes applied to the same element, being `height` not inherited. Happy to hear alternatives to this!

```
import { ConstrainedHeightContainer } from "./Routes/Conversation"

const ConstrainedContainer = styled(ConstrainedHeightContainer)`
  ...
`
```

Reply:
I have no clue how can I be helpful here... but the changes in this PR are enough to fix a bug, so unless anyone else has other ideas, I wouldn't hold this PR because of this.

I'm approving it for now and if we come up with ideas for improvements here, I would be fine having that submitted in a new PR

Reply:
I know was here before but I also don't like those magic numbers, I guess the right API in order to load more if we reach a certain part of the screen is the [Intersection Observer](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API) instead of crazy math with pixels, but the fix seems reasonable, gonna create a ticket to spike the Intersection Observer attempt 👍

Reply:
We have something similar in [Palette](https://github.com/artsy/palette/blob/main/packages/palette/src/utils/useSentinelVisibility.tsx) but the approach is different there, maybe we need a hook similar to this to load more or not if a sentinel div is reached out

Reply:
Thanks for the feedback and all the info!